rust/kcl-lib/src/execution/geometry.rs

```diff
@@ -737,6 +701,71 @@ impl SketchSurface {
     }
 }
 
+/// A Sketch, Face, or TaggedFace.
+#[derive(Debug, Clone, PartialEq)]
+pub enum SketchFaceOrTaggedFace {
+    /// Sketch.
+    Sketch(Box<Sketch>),
+    /// Face.
+    Face(FaceTag),
+    /// Tagged face.
+    TaggedFace(Box<TagIdentifier>),
+}
+
+impl SketchFaceOrTaggedFace {
+    /// Get the relevant id.
+    pub async fn id(
```

Comment:
I suggest renaming as `id_to_extrude` because we have a funny way of slowly adding more and more kinds of ID to our types haha.